week_4/practice/todo/protobuf/generate_cmd.txt

```diff
@@ -0,0 +1,3 @@
+protoc -I. -I$GOPATH/src -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --go_out=plugins=grpc:. *.proto
```

Comment:
Not sure the rationale behind this file. However, this command was added to `https://github.com/xuanit/testing/blob/8991b95d119da67714438f85c432e4424581062a/todo/pb/doc.go#L1`.
Whenever, we run `go generate` it's executed.

Reply:
this save my date.Make my project cleaner.